### PR TITLE
added cv luis

### PR DESCRIPTION
**Pull request by quinwacca**

### CODE REVIEW

File: staff/luis-garrido/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (nicopixel) [inline staff/luis-garrido/cv/index.html]: the language should be "es"

File: staff/luis-garrido/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+</head>
+
+<body style="max-width: 800px">
```
Comment (nicopixel) [inline staff/luis-garrido/cv/index.html]: without styles please! juuuurr

File: staff/luis-garrido/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    </header>
+
+    <h2>Professional Profile</h2>
+    <hr />
+    <p>In collaboration with others at Marvel—particularly co-writer/artists Jack Kirby and Steve Ditko—he co-created numerous popular fictional characters, including superheroes Spider-Man, the X-Men, Iron Man, Thor, the Hulk, the Fantastic Four, Black Panther, Daredevil, Doctor Strange, Scarlet Witch and Ant-Man. In doing so, he pioneered a more naturalistic approach to writing superhero comics in the 1960s, and in the 1970s he challenged the restrictions of the Comics Code Authority, indirectly leading to changes in its policies. In the 1980s he pursued development of Marvel properties in other media, with mixed results. Following his retirement from Marvel in the 1990s, he remained a public figurehead for the company, and frequently made cameo appearances in films based on Marvel characters, on which he received an honorary "executive producer" credit. Meanwhile, he continued independent creative ventures into his 90s.</p>
+    <h2>Awards</h2>
+    <hr />
+    <ul>
+        <li><time>1999</time> - The Will Eisner Award Hall of Fame</li>
```
Comment (nicopixel) [inline staff/luis-garrido/cv/index.html]: I think that you have to use the `datetime` attributte with a datetime value . For example: `<p>I have a date on <time datetime="2008-02-14 20:00">2008</time>.`
[source](https://www.w3schools.com/tags/tag_time.asp)

File: staff/luis-garrido/cv/index.html (hunk 2 above)
Comment (quinwacca) [reply staff/luis-garrido/cv/index.html]: Fixed!

File: staff/luis-garrido/cv/index.html (hunk 1 above)
Comment (quinwacca) [reply staff/luis-garrido/cv/index.html]: But the languaje is English.

File: staff/luis-garrido/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+</head>
+
+<body>
+    <header>
+            <h1>Stan Lee | Comic book writer, editor, publisher</h1>
+            <adress>
+                Location: <b>New York</b><br />
+                Telephone: <b><a href="tel:678678678">678 678 678</a></b><br />
+                Email: <b><a href="mailto:stan.lee@marvel.com">stan.lee@marvel.com</a></b><br />
+            </adress>
```
Comment (gcelma) [inline staff/luis-garrido/cv/index.html]: adress tag has 2 s --> address ;)

File: staff/luis-garrido/cv/index.html (hunk 1 above)
Comment (nicopixel) [reply staff/luis-garrido/cv/index.html]: totally true! forget about it 👍

File: staff/luis-garrido/cv/index.html (hunk 3 above)
Comment (nicopixel) [reply staff/luis-garrido/cv/index.html]: I've investigated a little more about the `<time>` tag and I've found this (page)[https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time]

In your line 27: `<time datetime="1999">1999</time>`

File: staff/luis-garrido/cv/index.html (hunk 4 above)
Comment (quinwacca) [reply staff/luis-garrido/cv/index.html]: Fixed! thanks!

File: staff/luis-garrido/cv/index.html (hunk 3 above)
Comment (quinwacca) [reply staff/luis-garrido/cv/index.html]: Fixed! Thanks for the info, Nico.

**Pull request merged**